README.md

```diff
@@ -315,6 +315,8 @@ debug(a, b)
 debug(a, b, merge=True)
 # DEBUG: a='value', b=<object object at 0x2b70580e5f20>
 debug(a, repr=False, prefix='') # a=value
+# also debug an expression
+debug(a+a, vars_only=False) # DEBUG: a+a='valuevalue'
```

Comment:
I think typing in vars_only=False looks like a pain. Why not make it the default?

Reply:
Sounds good.